specs/experimental/security-council-safe.md

```diff
@@ -77,6 +79,18 @@ interface DeputyGuardianModule {
 For simplicity, the `DeputyGuardianModule` module does not have functions for updating the `safe` and
 `deputyGuardian` addresses. If necessary these can be modified by swapping out with a new module.
 
+### Deputy Guardian Module Security Properties
+
+The following security properties must be upheld by the `DeputyGuardianModule`:
+
+1. The module must correctly enforce access controls so that only the Deputy Guardian can call state
+   modifying functions.
```

Comment:
Should we specify that these "state modifying functions" are all external to the Safe, and the deputy guardian should not be able to modify the state of the Safe itself or any of its extensions?

Reply:
Or more narrow: DGM only allows the deputy guardian to perform some or all of the actions that the regular guardian can take. It must not enable the deputy guardian to perform actions the guardian cannot perform

Reply:
Interesting. My intent here was to just to indicate that all state changing functions on the DGM itself should have auth checks so that only the DeputyGuardian could call them. 

Based on your comment I've expanded this item into 3 which I think aligns with both of our intents, except the 'some or all' part, ie. I don't think we would want a future situation where there exist a Guardian authed function which the `DeputyGuardian` cannot also call.

```
1. The module must correctly enforce access controls so that only the Deputy Guardian can call state
   modifying functions on the `DeputyGuardianModule`.
1. The module must be able to cause the Safe to make calls to all of the functions which the
   Guardian role is authorized to make.
1. The module must not be able to cause the Safe to make calls to functions which the Guardian role
   is not authorized to make.
```